sounddevice.py

```diff
@@ -2289,14 +2289,60 @@ def reset(self):
         vars(self).clear()
         self.__init__()
 
+
 if not hasattr(_ffi, 'I_AM_FAKE'):
     # This object shadows the 'default' class, except when building the docs.
     default = default()
 
 
 class PortAudioError(Exception):
-    """This exception will be raised on PortAudio errors."""
+    """This exception will be raised on PortAudio errors.
+
+    Parameters
+    ----------
+    msg : str, optional
+        User error message.
+    code : int, optional
+        PaErrorCode value.
+    hosterror_info : tuple, optional
```

Comment:
@mgeier Note that I expect the tuple described here and not the object returned by GetLastHostErrorInfo